setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
```

Comment:
What is the purpose of setuptools?

Reply:
setuptools is the built-in python utility for packaging projects